src/demo.js

```diff
@@ -5,35 +5,21 @@ import {
     annotation_display, parse_translations, create_context, format_messages
 } from './fluent';
 
-const translations = `// Try editing the translations here!
+const translations = `
+hello-user = Hello, { $user }!
```

Comment:
This new line makes all errors report line numbers which look off by one. Can you try putting the whole editor into a div which has the gray background and some padding?

Reply:
Feel free to do this in a followup but I'd like to see this fixed.